### PR TITLE
Homework#3 is done. 

### PR DESCRIPTION
**Pull request by OleksUMD**
All tests are passed successfully. Code was checked by flake8

### CODE REVIEW

File: python_functionality/homework.py [hunk 1]
```diff
@@ -11,45 +11,61 @@ def task_1_fix_names_start_letter(data: DT) -> DT:
...
 
     Examples:
-        fix_names_start_letters([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}])
+        fix_names_start_letters([{'name': 'Alex', 'age': 26},
+                                {'name': 'denys', 'age': 89}])
         >>> [{'name': 'Alex', 'age': 26}, {'name': 'Denys', 'age': 89}]
     """
-    pass
+    for dict_instance in data:
+        if dict_instance.get('name', None):
```
Comment (dimigor) [inline python_functionality/homework.py]: you do not need to add the second parameter, 'None' is already the default

File: python_functionality/homework.py [hunk 2]
```diff
@@ -11,45 +11,61 @@ def task_1_fix_names_start_letter(data: DT) -> DT:
...
+                            {'name': 'denys', 'age': 89}], 'age')
         >>> [{'name': 'Alex'}, {'name': 'denys'}]
     """
-    pass
+    for dict_instance in data:
+        for key in redundant_keys:
+            if dict_instance.get(key, None):
+                del dict_instance[key]
+
+    return data
```
Comment (dimigor) [inline python_functionality/homework.py]: try to solve tasks using dict/list comp

File: python_functionality/homework.py (hunk 1 above)
Comment (OleksUMD) [reply python_functionality/homework.py]: Fixed.

File: python_functionality/homework.py (hunk 2 above)
Comment (OleksUMD) [reply python_functionality/homework.py]: Done!

**Review by dimigor — APPROVED** [review]

**Pull request merged**